app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
```

Comment:
[REQUIRED] Kindly ensure that your foreign key is author_id so that it matches the ERD diagram as required.
![image](https://github.com/mumbereh/Blog_App/assets/62938735/fe82ec62-3395-4d69-89c6-9268a540067c)